dockerfiles/kind-images

```diff
@@ -50,8 +50,9 @@ COPY <<EOF /data/images.json
 EOF
 
 RUN lvh version
+ARG PKG_REPO
```

Comment:
maybe in those Dockerfiles, we coiuld provide a default value/or document this so people aren't lost why it exists when looking at it later on. Just put "unstable" as default otherwise